no-computer.md

```diff
@@ -11,41 +11,87 @@ _What need is met by your idea? Who are you helping? What is that person's
+The need met is learning about patterns and shapes. Additionally, there is
+a need of visualizing a pattern before using it. Before wooden blocks, there
+was a difficult experience of seeing how the shapes could fit together and
+understanding how angles worked in different shapes. These blocks give a
+tactile, visual way of learning about patterns and angles._
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_This language is appropriate because it does not involve drawing, which
+can be inprecise, or imagining, which can leave people unsatisfied. It 
+addresses the need by giving them wooden blocks, which have all been cut
+to be certain shapes. This lets them interact with the blocks and try all
+different types of blocks to see which ones fit._
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_I love this idea because there are only so many ways that the
+blocks can interact with each other that does not leave any blank space
+in between them. It's such an interesting way to introduce geometry and
+learn about the angles of the shapes when the math might still be too
+hard. I came up with this idea by thinking about the argument we discussed
+in class that the marble run was a DSL. I led me to thinking about how
+other childhood toys could be DSLs, which made me think about the wooden
+shape blocks._
 
 ### Domain
 _Describe the project's domain in five words._
 
+_Shapes coming together, making patterns_
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+_People would be able to interact with the language by phyically grabbing
+the blocks and trying them out in the program. This is the right way
+to interact with the problem domain because instead of asking people to
+imagine what happens when a square's edge hits a diamond's edge, they
+can physically grab the square and see what happens. This can be far
+less confusing for a user, espcially as patterns get larger._
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+_I'm not completely certain how a program would run for this DSL because there's
+no point where you stop and test it, but I think that it would run each time that
+a user would put a new piece on or move a piece. The errors would be seen visually
+by a space in between the blocks. Although these are not terrible errors, when
+trying to create patterns or sets of blocks that can be repeated and used for
+other domains such as creating art, this blank space cannot be there. So, the 
+user would be able to visually see when they have made an error. The program would
```

Comment:
This is a really cool take on error messages. The "program" fails and the error is shown immediately, when pieces literally don't fit together.